packages/big-design/src/components/Flex/Item/styled.tsx

```diff
@@ -6,7 +6,8 @@ import { withFlexedItems } from '../withFlex';
 
 import { FlexItemProps } from './Item';
 
-export const StyledFlexItem = styled(Box)<FlexItemProps>`
+// Remove the `forwardedAs` manual prop definition when @types get updated
```

Comment:
🍹Can we prefix these with `TODO:` so we can easily search for them?
```suggestion
// TODO: Remove the `forwardedAs` manual prop definition when @types get updated
```